fiware-region-sanity-tests/commons/neutron_operations.py

```diff
@@ -120,6 +129,53 @@ def create_router(self, router_name, external_network_id=None):
         self.logger.debug("Created router %s", neutron_network_response['router']['id'])
         return neutron_network_response['router']
 
+    def add_router_interface(self, router_id, subnetwork_id):
+        """
+        Adds a new port to the router, linked to the given subnet
+        :param router_id: Rotuer ID
+        :param subnetwork_id: Subnetwork ID to be linked to the router
+        :return: Port id (String)
+        """
+        add_interface_body = self.__build_body_add_interface__(subnetwork_id)
+        response_body = self.client.add_interface_router(router=router_id, body=add_interface_body)
+        self.logger.debug("Linked subnet '%s' to router %s", subnetwork_id, router_id)
+        return response_body['port_id']
+
+    def delete_router_interface(self, router_id, subnetwork_id):
```

Comment:
Idem, but not "remove_interface_router"... I'd use "delete_interface_router" because "delete_" is being used in the rest of operations...